internal/distro/image_config.go

```diff
@@ -20,6 +20,8 @@ type ImageConfig struct {
 	DefaultTarget       string
 	Sysconfig           []*osbuild2.SysconfigStageOptions
 
+	GPGKeyFiles []string
```

Comment:
I would suggest to add "RPM" to the variable name to make it obvious what type of GPG keys these are.

In addition, you need to extend the `InheritFrom` method due to this new member.

Reply:
Good catch! We should enhance that to use reflection in a future version (and maybe switch it do clone and override ;) ;)

Reply:
_nods_

Reply:
I added a comment for now, can always rename it later if it collides with something.

Reply:
It probably won't collide, I would just find "RPM" in the variable name easier to understand when one uses the structure, compared to a comment in its definition. But I agree with you and think that the current state is OK to merge.